src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { nowUOrange } from '../../assets';
+import {
+    topNavLinks,
+    bottomNavLinks,
+    mobileNavLinks,
+} from '../../utils/constants';
+import NavLinkSection from './navLinkSection/NavLinkSection';
+import NavDropdown from './navDropdown/NavDropdown';
+import classes from './Navbar.module.scss';
+
+const Navbar = () => {
+    const [displayNavDropdown, setDisplayNavDropdown] = useState(false);
+    const toggleDisplayNav = (...args) => {
+        if (args.length === 0) {
+            setDisplayNavDropdown((prevState) => !prevState);
+        }
+        if (args.length > 0) {
+            const direction = args[0];
+            setDisplayNavDropdown(direction);
+        }
```

Comment:
Maybe something like this - What if someone gives a whole load of random arguments of a single argument that is not a boolean?
```suggestion
    const toggleDisplayNav = ({bool open}) => {
        openState = (prevState) => !prevState 
        if (open) {
           openState = open
        }
        setDisplayNavDropdown(openState)
```

Then when calling this function call with `toggleDisplayNav(open=true)` or `toggleDisplayNav()`